crates/linker/src/macho.rs

```diff
@@ -58,8 +63,8 @@ struct Metadata {
     surgeries: MutMap<String, Vec<SurgeryEntry>>,
     dynamic_symbol_indices: MutMap<String, u64>,
     roc_symbol_vaddresses: MutMap<String, u64>,
-    exec_len: u64,
-    load_align_constraint: u64,
+    exec_len: usize,
+    load_align_constraint: usize,
```

Comment:
Doesn't this get serialized? Will usize mess that up?

Reply:
Mhm, that's a good point! I will rethink all instances of usize in metadata and convert them to explicit 32- or 64-bit types. Thanks for pointing that out to me!